src/styles/card.scss

```diff
@@ -0,0 +1,82 @@
+* {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    width: 160px;
+    height: 134px;
+    background-size: 100% 100%;
+    background-position: center;
+    background-repeat: no-repeat;
+    margin-bottom: 40px;
+    align-self: center;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $second-text-color;
+    margin-bottom: 16px;
+  }
+}
+
+.card-price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+
+  &__title {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $second-text-color;
+  }
+
+  &__value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+}
+
+.card-button {
+  height: 40px;
+
+  background-color: $button-text-color;
+  color: $background-button-color;
```

Comment:
Ensure that the SCSS variables `$button-text-color` and `$background-button-color` are defined in a separate variables file as per the task requirements.